pkg/health/client/client.go

```diff
@@ -0,0 +1,23 @@
+package client
+
+import (
+	"context"
+
+	"github.com/square/p2/pkg/health"
+)
+
+type HealthRequest struct {
+	Url      string
```

Comment:
any reason you didn't want to use the `url.URL` type? just easier to swap out protocols, etc?

Reply:
I initially used `url.URL` but there seems to be more usage of the `string` type in the code I was looking at and using. The `url.URL` type also has a `protocol` field which feels redundant (and sometimes harder to work with)